src/ProductsXmlFeed.php

```diff
@@ -414,7 +414,22 @@ private static function get_taxonomies( $product_id ) {
 	}
 
 	/**
-	 * Fetch shipping object.
+	 * Get locale currency decimals
+	 */
+	private static function get_currency_decimals() {
+		$locale_info = include WC()->plugin_path() . '/i18n/locale-info.php';
```

Comment:
Maybe move the `include` in the conditional as well so that it doesn't load up the array when not needed.